src/hoi4_world/states/hoi4_states_converter.cpp

```diff
@@ -17,7 +17,7 @@
 namespace
 {
 
-constexpr float MAX_FACTORY_SLOTS = 12.0F;
+constexpr int MAX_FACTORY_SLOTS = 12;
```

Comment:
12 is iffy. You fill up to 12 which is max, sure, but in vanilla how many states actually start at 12? I'd cut this by half. In fact, I'd probably cut entire industry by half. Not 75k per industry but 150k. Even that is questionable if you want to hit vanilla starting scales of 20-50 per major, and probably some sort of scaling will be required.

Even if you own entire europe as Rome, and all the industry inside, starting at 500 civs and 800 mils is just meaningless as you outstrip resource availability. Better to actually scale to *50*.